README.rst

```diff
@@ -3,3 +3,8 @@ wow-item-expansion
 
 Displays the item's extension of origin in item's tooltip; so you can see easily from which expansion an object
 originate from.
+
+:warning: Deprecated in favor of https://www.curseforge.com/wow/addons/itemversion. And inferior.
```

Comment:
Is the "And inferior." part intended? It seems as residue from some previous version.

Reply:
My syntax. Bad. 😄